src/containerapp/azext_containerapp/_params.py

```diff
@@ -126,6 +132,12 @@ def load_arguments(self, _):
         c.argument('system_assigned', options_list=['--mi-system-assigned'], help='Boolean indicating whether to assign system-assigned identity.', action='store_true')
         c.argument('user_assigned', options_list=['--mi-user-assigned'], nargs='+', help='Space-separated user identities to be assigned.')
 
+    with self.argument_context('containerapp env certificate upload') as c:
+        c.argument('certificate_identity', options_list=['--certificate-identity', '--identity'], 
+                   help='Resource ID of a managed identity to authenticate with Azure Key Vault, or System to use a system-assigned identity.', is_preview=True)
+        c.argument('certificate_key_vault_url', options_list=['--certificate-key-vault-url', '--key-vault-url'], 
+                   help='The URL pointing to the Azure Key Vault secret that holds the certificate.', is_preview=True)
```

Comment:
these code is unnecessary because you already have same argument declaration in 'containerapp env'

Reply:
seems the parameter will only take effect in commands but not for subgroup, so need to define them